### ISSUE TITLE
Aria-level setting ignored for component item aria level (always announced as heading level 2)

### ISSUE THREAD

**Issue by RMurray89** [cgkineo/adapt-tabs#33]
Hi,

I believe there is an issue with the setting of the aria level on this component. The item title always appears to be announced as Level 2.

I've tried two ways to set the the correct aria-level.

1) Component Item element ARIA level setting in Project Configuration of the authoring tool. I've set this to level 4, and although the code shows level 4 

![image](https://user-images.githubusercontent.com/3612643/169076881-b411a3f1-a704-4b99-8eba-31106993837d.png)

![image](https://user-images.githubusercontent.com/3612643/169075977-8f09ef28-fc54-4e9b-bcc2-09c6cf7471d0.png)

The screenreader will still announce it as a level 2.

2) Attempted to change aria-level in component settings but no effect.

The level is not logical for my content order and I need to change it, any help appreciated.

Thanks
Ross

**Comment by oliverfoster** [comment]
It should be aria-level='4'

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role

**Comment by oliverfoster** [comment]
https://github.com/cgkineo/adapt-tabs/releases/tag/v4.0.1

**Comment by RMurray89** [comment]
> It should be aria-level='4'
> 
> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role

Ah of course, I blindly didn't actually notice that. Thank for the quick fix.
